var/spack/repos/builtin/packages/crmc/package.py

```diff
@@ -50,5 +50,6 @@ def cmake_args(self):
         ]
         if self.spec.satisfies("@1.6.0:"):
             args.append("-D__HIJING__=ON")
-        args.append("-DCMAKE_Fortran_FLAGS=-fallow-argument-mismatch")
+        if self.spec.satisfies("%gcc@9:") and self.spec.satisfies("%clang@13:"):

```

Comment:
Shouldn't this be `if self.spec.satisfies("%gcc@9:") or self.spec.satisfies("%clang@13:"):` (`or` instead of `and`)?

Reply:
Oops, thanks!